eclipse/che
# Remove Keycloak references

### What does this PR do?
https://github.com/eclipse/che/issues/21175

### Screenshot/screencast of this PR



### What issues does this PR fix or reference?



### How to test this PR?

After removing Keycloak references, still the test cases should work. Executed below commands for testing purpose
`
> export TS_SELENIUM_BASE_URL=https://192.168.59.165.nip.io
> export TS_SELENIUM_USERNAME=che@eclipse.org
> export TS_SELENIUM_PASSWORD=admin
> export USERSTORY=VscodeYamlPlugin
> npm run test-plugin`
Attaching Yaml plugin execution logs : [yamlPlugin_ExecutionLogs.txt](https://github.com/eclipse/che/files/8197091/yamlPlugin_ExecutionLogs.txt)

> export USERSTORY=Go
> npm run test-all-devfiles
Execution logs for Go spec file:  [Go_spec_execution_logs.txt](https://github.com/eclipse/che/files/8197055/Go_spec_execution_logs.txt)

#### Verify the downstream code with upstream

> 


### PR Checklist

[As the author of this Pull Request I made sure that:](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#pull-request-template-and-its-checklist)


### Reviewers

Reviewers, please comment how you tested the PR when approving it.
